airflow/www/templates/airflow/dags.html

```diff
@@ -137,7 +137,11 @@ <h2>DAGs</h2>
 
             <!-- Column 5: Dag Owners -->
             <td>
-              {{ dag.owners }}
+              <a class="label label-default"
+                 href="?search={{ dag.owners }}"
+                 style="margin: 3px;">
+                 {{ dag.owners }}
```

Comment:
I wonder if we should not display a separate link for each person.

Reply:
How do you think we should do this ? Also why not links, I am asking because everything else is a link 😅

Reply:
^^ @mik-laj

Reply:
````html
                  {% for owner in dag.owners.split(",") %}
                  <a class="label label-default"
                     href="?search={{ owner | trim }}"
                     style="margin: 3px;">
                     {{ owner | trim }}
                  </a>
                  {% endfor %}
````
<img width="428" alt="Screenshot 2020-11-08 at 01 08 57" src="https://user-images.githubusercontent.com/12058428/98453942-130a8b80-215f-11eb-8aee-ee1998343fb8.png">

What do you think aboiut it?

Reply:
Before:
<img width="339" alt="Screenshot 2020-11-08 at 01 11 24" src="https://user-images.githubusercontent.com/12058428/98453972-582ebd80-215f-11eb-9c07-de35bca0f35b.png">

Reply:
Ah yes I think I understood the question wrong, this does make more sense